marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                    "*"
+                ],
+                "title": "Cluster status",
+                "title_align": "center",
+                "title_size": "16",
+                "type": "check_status"
+            },
+            "id": 8755180811202380,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 24,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "check": "marklogic.forest.health",
+                "group_by": [],
+                "grouping": "cluster",
+                "tags": [
+                    "*"
+                ],
+                "title": "Forests health",
+                "title_align": "center",
+                "title_size": "16",
+                "type": "check_status"
+            },
+            "id": 2200143632431740,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 40,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "check": "marklogic.database.health",
+                "group_by": [],
+                "grouping": "cluster",
+                "tags": [
+                    "*"
+                ],
+                "title": "Databases health",
+                "title_align": "center",
+                "title_size": "16",
+                "type": "check_status"
+            },
+            "id": 8505096373419124,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 56,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "requests": [
+                    {
+                        "q": "top(sum:marklogic.forests.storage.disk_size{*} by {forest_name}, 10, 'last', 'desc')"
+                    }
+                ],
+                "time": {},
+                "title": "Total size of forests on disk",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "toplist"
+            },
+            "id": 7675935468546344,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 73,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Forests summary",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 2288121390055484,
+            "layout": {
+                "height": 5,
+                "width": 38,
+                "x": 73,
+                "y": 0
+            }
+        },
+        {
+            "definition": {
+                "background_color": "blue",
```

Comment:
I think we should make this a category (rather than a subcategory) and add a "#" before "MarkLogic logs"